docs/commands/index.md

```diff
@@ -6,11 +6,11 @@ tocMaxDepth: 2
 
 Documentation commands:
 
-- [`preview-docs`](preview-docs.md) Preview API reference docs for the specified API description.
-- [`build-docs`](build-docs.md) Build API description into an HTML file.
 - [`preview`](preview.md) Start a local preview of a Redocly project with one of the product NPM packages.
 - [`translate`](translate.md) Generate translation keys for a Redocly Realm, Reef, or Revel project.
 - [`eject`](eject.md) Eject and modify components from the core theme in a Redocly Realm, Reef, or Revel project.
+- [`preview-docs`](preview-docs.md) Preview API reference docs for the specified API description. Deprecated in the near future.
```

Comment:
```suggestion
- [`preview-docs`](preview-docs.md) Preview API reference docs for the specified API description.
```

Reply:
I do not think this statement is helpful.

Reply:
How do we indicate that people shouldn't use this command?

Reply:
I guess if we don't want them to use it perhaps we shouldn't include it in the documentation at all. Is there a replacement for it?

Reply:
The `preview` command instead of `preview-docs`. Maybe instead I'll add an admonition in that command. I'll do that as a separate PR as it's unrelated to this PR and remove this from this PR. 👍